packages/neo-one-smart-contract-compiler/src/compile/expression/BinaryExpressionCompiler.ts

```diff
@@ -91,9 +91,11 @@ export default class BinaryExpressionCompiler extends NodeCompiler<
     const pushValueOptions = sb.pushValueOptions(options);
     switch (kind) {
       case SyntaxKind.EqualsToken:
+        // Tested
```

Comment:
Not a huge fan of these comments, but since we can't really get code coverage reports it's probably better to leave them in.